internal/services/recoveryservices/backup_protected_vm_resource.go

```diff
@@ -145,27 +140,23 @@ func resourceRecoveryServicesBackupProtectedVMCreateUpdate(d *pluginsdk.Resource
 	protectionState, ok := d.GetOk("protection_state")
 	protectionStopped := strings.EqualFold(protectionState.(string), string(protecteditems.ProtectionStateProtectionStopped))
 	requireUpdateProtectionState := ok && protectionStopped
-	skipNormalUpdate := protectionStopped && !d.IsNewResource()
-
-	// stopped protected item has no `backup_policy_id`, though we can update it before stopping we can not read it.
-	if !skipNormalUpdate {
-		resp, err := client.CreateOrUpdate(ctx, id, item)
-		if err != nil {
-			return fmt.Errorf("creating/updating Azure Backup Protected VM %q (Resource Group %q): %+v", protectedItemName, resourceGroup, err)
-		}
 
-		operationId, err := parseBackupOperationId(resp.HttpResponse)
-		if err != nil {
-			return fmt.Errorf("issuing creating/updating request for %s: %+v", id, err)
-		}
+	resp, err := client.CreateOrUpdate(ctx, id, item)
+	if err != nil {
+		return fmt.Errorf("creating Azure Backup Protected VM %q (Resource Group %q): %+v", protectedItemName, resourceGroup, err)
```

Comment:
Can we not use the id for the error message here?
```suggestion
		return fmt.Errorf("creating %s: %+v", id, err)
```